migration/v0.36/apply/11-kube-prometheus-stack-crds.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
+
+source "${ROOT}/scripts/migration/lib.sh"
+
+run() {
+  case "${1:-}" in
+  execute)
+    if [[ "${CK8S_CLUSTER}" =~ ^(sc|both)$ ]]; then
```

Comment:
I would like to add a condition and run this migrations steps only when necessary

Reply:
From [guide:](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
> With Helm v3, CRDs created by this chart are not updated by default and should be manually updated. Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).

Due to this, a migration will always be needed technically when upgrading kube-prometheus-stack.

Reply:
yes, but if we already run the upgrade kube-prometheus-stack version, do we still want to apply the CRDs and re-do the upgrade?

Reply:
Fixed by adding "app!=prometheus" to skipped in "80-apply.sh" .